web_poet/testing/fixture.py

```diff
@@ -232,11 +262,19 @@ def save(
         storage = SerializedDataFileStorage(fixture.input_path)
         storage.write(serialized_inputs)
 
-        with fixture.output_path.open("w") as f:
-            f.write(cls.item_to_json(item))
+        if item is not None:
+            with fixture.output_path.open("w") as f:
+                f.write(cls.item_to_json(item))
 
         if meta:
             with fixture.meta_path.open("w") as f:
                 json.dump(meta, f, ensure_ascii=False, indent=4)
 
+        if exception:
+            with fixture.exception_path.open("w") as f:
+                exc_data = {
+                    "type_name": get_fq_class_name(type(exception)),
+                }
+                json.dump(exc_data, f, ensure_ascii=False, indent=4)
```

Comment:
Is there a reason we have two ways/formats to store an exception - here and in https://github.com/scrapinghub/web-poet/pull/163/files#diff-fddeae0fca9e0d2911f103f05ecd5c10af4494f7b24fad04b214aae96427128cR127? Would it make sense to refactor it?